### DISCUSSION TITLE
One-way FSI in the MPHYS during the aero-structural optimization

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#357] [General]
Dear Prof. Ping, 
As far as I know, the FSI used in Mphys during the aero-structural optimization is a two-way FSI. How can I change it to one-way FSI only? Basically, I want only transfer pressure from the aerodynamic side to the structural side but not the displacement from the structural side to aerodynamic side. 
How can I make it possible to run? 
Thanks a lot,

**Comment by friedenhe** [comment]
No, there is no easy way to set up a one-way FSI optimization in the run script. You have to hack into mphys to make that happen...
